datafusion/functions/src/utils.rs

```diff
@@ -170,7 +171,8 @@ pub mod test {
                     }
                     else {
                         // invoke is expected error - cannot use .expect_err() due to Debug not being implemented
-                        match func.invoke_with_args(datafusion_expr::ScalarFunctionArgs{args: $ARGS, number_rows: cardinality, return_type: &return_type.unwrap()}) {
+                        #[allow(deprecated)]
```

Comment:
likewise here -- invoke_with_args

Reply:
A lot of tests use this macro and pass in &[], so it quite a big change.